### PR TITLE
Created New Error Messages for Invalid and Non Yaml Playbook Files

### PR DESCRIPTION
**Pull request by humzam96**
##### SUMMARY
Upon Project Update a playbooks list was updated and only includes files that were valid. We added 2 new lists for invalid playbooks (playbooks that fail to match the regular expression in awx/main/utils/ansible.py) and non yaml files. This allows for more specific validation errors when validating in JobOptionsSerializer in awx/api/serializers.py. We tried looking for a solution to avoid creating a new migration, if there is a different suggestion we can try that too.

related #6842 

##### ISSUE TYPE
 - Feature Pull Request

##### COMPONENT NAME
 - API

##### AWX VERSION
```
awx: 12.0.0
```

### CODE REVIEW

File: awx/main/models/projects.py [hunk 1]
```diff
@@ -431,13 +489,16 @@ def notification_templates(self):
         if self.organization is not None:
             error_notification_templates = set(error_notification_templates +
                                                list(base_notification_templates
-                                                    .filter(organization_notification_templates_for_errors=self.organization)))
+                                                    .filter(
```
Comment (wenottingham) [inline awx/main/models/projects.py]: There's a bunch of unrelated formatting changes in this PR. Please remove them.

Comment (humzam96) [reply awx/main/models/projects.py]: Thanks, for some reason those changes were recommended when tested on a local build, I just undid them and the Zuul tests passed.

**Comment by humzam96** [comment]
Do you have any recommendations for updating this solution if it's not something that can be accepted?

File: awx/main/migrations/0118_v400_invalid_playbooks.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Generated by Django 2.2.11 on 2020-06-04 03:55
```
Comment (ryanpetrello) [inline awx/main/migrations/0118_v400_invalid_playbooks.py:1]: This migration is out of date and needs to be regenerated to point at the latest.

**Comment by ryanpetrello** [comment]
Hey @humzam96,

This PR seems considerably out of date given code and database schema changes over the past 9 months.  If you're still interested in contributing this, please open another PR.

Thanks.
